### DISCUSSION TITLE
Cannot read property 'reduce' of undefined

### DISCUSSION THREAD

**Discussion by DonnaTelloo** [react-hook-form/react-hook-form#4975] [Q&A] [answered]
I have built form with yup schema. When I click submit button throws error
![image](https://user-images.githubusercontent.com/23365416/116228527-412e5880-a70a-11eb-95de-5e83991d2623.png)

**Comment by bluebill1049** [comment]
we need more context and details, which version and maybe reproduce that in a codesandbox?

**Comment by leapful** [comment, marked as answer]
@DonnaTelloo

It might be the same issue with https://github.com/react-hook-form/react-hook-form/discussions/4980

Please check your custom yup validation to see if there is any unexpected while performing validation (like accessing undefined value).
